packages/react/src/jsx/ReactJSXElement.js

```diff
@@ -16,7 +16,19 @@ import {
 } from 'shared/ReactSymbols';
 import {checkKeyStringCoercion} from 'shared/CheckStringCoercion';
 import isArray from 'shared/isArray';
-import {disableDefaultPropsExceptForClasses} from 'shared/ReactFeatureFlags';
+import {
+  disableDefaultPropsExceptForClasses,
+  debugInfoLimitsResetIntervalMs,
+  debugTaskLimit,
+  ownerStackLimit,
+} from 'shared/ReactFeatureFlags';
+
+let recentlyCreatedOwnerStacks = 0;
+let recentlyCreatedDebugTasks = 0;
+setInterval(() => {
```

Comment:
Can we move the counter and timer to isomorphic react?

It would also be good to reset the counter every commit (and reset the timer too, but maybe not start it until the next render).

Reply:
Do we not mean `react` with "isomorphic react"?

Reply:
This interval will be duplicated across `jsx` and `createElement` since they're in separate bundles. So it's two.

However, that seems fine. If you have a run away number of elements you'll have that with either `createElement` or `jsx`.

Better than leaking this info outside the module and having to read it from an object.

The reason to do isomorphic would be if it was reset using commit or something in each renderer. Although if you have a bunch of frequent commits, maybe it's not so much better.

It does seem weird to have a this running all the time even when React isn't rendering though.

Also, this needs to be DEV-only gated and feature detected if `setInterval` exists.

Reply:
> It does seem weird to have a this running all the time even when React isn't rendering though.

That would require adding a dedicated method to internals, right? Should we just share a method to reset with renderers and they decide when to reset (timer-based, commit based etc.) or should it just be a a simple start/stop interval?